src/sentry/profiles/task.py

```diff
@@ -949,3 +954,14 @@ def _push_profile_to_vroom(profile: Profile, project: Project) -> bool:
         reason="profiling_failed_vroom_insertion",
     )
     return False
+
+
+def prepare_android_js_profile(profile: Profile):
```

Comment:
If I understand correctly, we're copying those fields to be able to symbolicate properly via the `_symbolicate_profile` function. Can we remove them after since we don't need them for rendering?